test/Validate.Capability.cpp

```diff
@@ -127,7 +127,9 @@ const vector<string>& AllCapabilities() {
     "StorageImageReadWithoutFormat",
     "StorageImageWriteWithoutFormat",
     "MultiViewport",
-    "SubgroupDispatch"};
+    "SubgroupDispatch",
+    "NamedBarrier",
```

Comment:
NamedBarrier is off-topic for this CL.   Ok to have in this CL if you update the commit message to mention it as a drive-by addition.

Also, I suspect you'll want another CL for validator tests of the PipeStorage capability.